remotebackup.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
+
+#This script connects  to a remote server via ssh, creates a .tar.gz backup, and uses scp to secure copy it to the host computer. 
```

Comment:
It's because I'm too verbose in  the comment, right?  I'll simplify the language.  Thanks!